tests/integration/test_end_to_end_dcp.py

```diff
@@ -8,11 +8,11 @@
 from ingest.importer.submission import Submission
 
 from tests.wait_for import WaitFor
-from ..utils import Progress, Timeout
-from ..cloudwatch_handler import CloudwatchHandler
-from ..data_store_agent import DataStoreAgent
-from ..dataset_fixture import DatasetFixture
-from ..dataset_runner import DatasetRunner
+from tests.utils import Progress, Timeout
```

Comment:
I think these changes are not needed once you are done testing in your own environment. Maybe remove them from the PR and keep these changes local/untracked?